index.d.ts

```diff
@@ -17,33 +17,61 @@ export interface SnapshotOptions {
 	id?: string;
 }
 
+// When an assert fail, the message string will be shown in the error
 export interface Assertions {
+	// Assert that actual is deep equal to expected
 	deepEqual<ValueType = any>(actual: ValueType, expected: ValueType, message?: string): void;
+	// Failing assertion
 	fail(message?: string): void;
+	// Assert that actual is equal to false
 	false(actual: any, message?: string): void;
+	// Assert that actual is falsy (false, 0, '', "", null, undefined, NaN)
 	falsy(actual: any, message?: string): void;
+	// Assert that error is falsy
 	ifError(error: any, message?: string): void;
+	// Assert that actual is equal to expected (use deepEqual for objects/arrays)
```

Comment:
Perhaps "strictly equal". Using this with objects or arrays can be a valid use case so the reference to `deepEqual` seems distracting. There's also edge cases with negative zero and NaN but that's perhaps too detailed for inline documentation 😉